SocialCareCaseViewerApi.Tests/V1/Controllers/MashResident/UpdateMashResidentTests.cs

```diff
@@ -44,5 +45,37 @@ public void WhenSuccessfullyUpdateActionReturns204AndUpdatedMashResident()
             response?.StatusCode.Should().Be(200);
             response?.Value.Should().BeEquivalentTo(updatedMashResident);
         }
+
+        [Test]
+        public void WhenMashResidentUseCaseUpdateMashResidentThrowsPersonNotFoundExceptionReturnsBadRequest()
+        {
+            var matchingPersonId = _faker.Random.Long(3, 4);
```

Comment:
As I've recently been reminded by @Renny4Real (after a comment where I suggested a similar solution with a random number), there is a (very small) chance the ID we want to test with might coincidentally match the ID of the person we setup in the database for the test. So maybe adding a number to the ID itself (as was done in the integration tests that have now moved here) might exclude that chance..?

Reply:
Long - Generate a random long between **MinValue** and **MaxValue**. From the Bogus source [here](https://github.com/bchavez/Bogus).

`_fakeMashResidentId = _faker.Random.Long(1, 2);`

It can become either 1 or 2

`matchingPersonId = _faker.Random.Long(3, 4);`

It can become either 3 or 4.

Unless Faker will behave unexpectedly there is 0 chance of accidental match.